client/components/PetTypesPage.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react'
+import PetTypesItem from './PetTypesItem'
+
+const PetTypesPage = (props) => {
+    const [petTypes, setPetTypes] = useState([])
+    
+    const getPetTypes = async () => {
+        try {
+            const response = await fetch("/api/v1/pet-types")
+            if (!response.ok) {
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw (error)
+            }

```

Comment:
The indentation here should only be 2 spaces. Please refer to Martin's post on how to adjust your vscode settings.